client/state/plugins/wporg/selectors.js

```diff
@@ -24,4 +24,36 @@ const isFetched = function( state, pluginSlug ) {
 	return !! plugin.fetched;
 };
 
-export default { getPlugin, isFetching, isFetched };
+const canFetchList = function( state, category, page, searchTerm ) {
```

Comment:
Yeah, you're right! let's add them in another PR to not further complicate this one :+1: